### PR TITLE
513 Add example(s) to `tt_at_path`

### LINKED ISSUE #513: Add example(s) to `tt_at_path`

### PR DESCRIPTION
**Pull request by BFalquet**
close #513 

add an example demonstrating the capbilities of `tt_at_path`

thank you for the review

### CODE REVIEW

File: R/tt_pos_and_access.R [hunk 1]
```diff
@@ -291,7 +319,7 @@ setMethod("tt_at_path<-", c(tt = "VTableTree", value = "VTableTree"),
 })
 
 ## this one removes the child at path from the parents list of children,
-## becuase thats how lists behave.
+## because that's how lists behave.
```
Comment (Melkiades) [inline R/tt_pos_and_access.R:322]: it is always better not have contraptions in written form (I know that it was as such already), but it is better to have `that is` in written form

File: R/tt_pos_and_access.R [hunk 2]
```diff
@@ -280,6 +280,34 @@ setMethod("tt_at_path", "VTableTree",
...
+#' 
+#' @examples 
+#' # Accessing sub table.
+#' lyt <- basic_table() %>%
+#'    split_cols_by("ARM") %>%
+#'    split_rows_by("SEX") %>%
+#'    split_rows_by("BMRKR2") %>%
+#'    analyze("AGE")
+#'
+#' tbl <- build_table(lyt, ex_adsl)
```
Comment (Melkiades) [inline R/tt_pos_and_access.R:292]:
```suggestion
#' tbl <- build_table(lyt, ex_adsl) %>% prune_table()
```

File: R/tt_pos_and_access.R [hunk 3]
```diff
@@ -280,6 +280,34 @@ setMethod("tt_at_path", "VTableTree",
...
+#' tt_at_path(tbl2, path = c("SEX", "F")) <- NULL
+#' tbl2
+#'
+#' # Setting sub table.
+#' lyt3 <- basic_table() %>%
+#'    split_cols_by("ARM") %>%
+#'    split_rows_by("SEX") %>%
+#'    analyze("BMRKR2")
+#'
+#' tbl3 <- build_table(lyt3, ex_adsl)
```
Comment (Melkiades) [inline R/tt_pos_and_access.R:306]:
```suggestion
#' tbl3 <- build_table(lyt3, ex_adsl) %>% prune_table()
```
Just better for clarity

**Review by Melkiades — APPROVED** [review]
Lgtm! Thank you :)

File: R/tt_pos_and_access.R [hunk 4]
```diff
@@ -280,6 +280,34 @@ setMethod("tt_at_path", "VTableTree",
 
 #' @export
 #' @rdname ttap
+#' 
+#' @examples
```
Comment (gmbecker) [inline R/tt_pos_and_access.R:284]: Please add text specifying that assigning `NULL` removes the subtable at the path, in addition to spelling that out in the example?

**Review by gmbecker — APPROVED** [review]

**Pull request merged**